tests/functional/spec/census/routing/carer.spec.js

```diff
@@ -0,0 +1,176 @@
+    startCensusQuestionnaire('census_household.json', true)
+    PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
+    HouseholdComposition.setFirstName('John').addPerson().setFirstName('Jane', 1).submit()
+    EveryoneAtAddressConfirmation.clickEveryoneAtAddressConfirmationAnswerYes().submit()
+    OvernightVisitors.setOvernightVisitorsAnswer(0).submit()
+    HouseholdRelationships.clickHouseholdRelationshipsAnswerHusbandOrWife().submit()
+    WhoLivesHereCompleted.submit()
+
+    // household-and-accommodation
+    TypeOfAccommodation.clickTypeOfAccommodationAnswerWholeHouseOrBungalow().submit()
+    TypeOfHouse.clickTypeOfHouseAnswerSemiDetached().submit()
+    SelfContainedAccommodation.clickSelfContainedAccommodationAnswerYesAllTheRoomsAreBehindADoorThatOnlyThisHouseholdCanUse().submit()
+    NumberOfBedrooms.setNumberOfBedroomsAnswer(3).submit()
+    CentralHeating.clickCentralHeatingAnswerGas().submit()
+    OwnOrRent.clickOwnOrRentAnswerOwnsOutright().submit()
+    NumberOfVehicles.setNumberOfVehiclesAnswer(2).submit()
+    HouseholdAndAccommodationCompleted.submit()
+
+    // household-member
+    HouseholdMemberBegin.submit()
+    DetailsCorrect.clickDetailsCorrectAnswerYesThisIsMyFullName().submit()
+    Over16.clickOver16AnswerYes().submit()
+    PrivateResponse.clickPrivateResponseAnswerNoIDoNotWantToRequestAPersonalForm().submit()
+    Sex.clickSexAnswerMale().submit()
+    DateOfBirth.setDateOfBirthAnswerDay(2).setDateOfBirthAnswerMonth(3).setDateOfBirthAnswerYear(1980).submit()
+    MaritalStatus.clickMaritalStatusAnswerMarried().submit()
+    AnotherAddress.clickAnotherAddressAnswerNo().submit()
+    InEducation.clickInEducationAnswerYes().submit()
+    TermTimeLocation.clickTermTimeLocationAnswerHereAtThisAddress().submit()
+    CountryOfBirth.clickCountryOfBirthEnglandAnswerEngland().submit()
+    Carer.clickCarerAnswerNo().submit()
+    expect(NationalIdentity.isOpen()).to.equal(true, 'Expecting go to national identity')
+  })
+
+  it('Given I am answering question 13. Do you look after, or give any help or support..., When I select -Yes 1-19 Hours A Week- as response, Then I am routed to 14. How would you describe your national identity?', function () {
+    startCensusQuestionnaire('census_household.json', true)
+    PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
+    HouseholdComposition.setFirstName('John').addPerson().setFirstName('Jane', 1).submit()
+    EveryoneAtAddressConfirmation.clickEveryoneAtAddressConfirmationAnswerYes().submit()
+    OvernightVisitors.setOvernightVisitorsAnswer(0).submit()
+    HouseholdRelationships.clickHouseholdRelationshipsAnswerHusbandOrWife().submit()
+    WhoLivesHereCompleted.submit()
+
+    // household-and-accommodation
+    TypeOfAccommodation.clickTypeOfAccommodationAnswerWholeHouseOrBungalow().submit()
```

Comment:
I think it will be useful to use the helper methods in the complete-sections.js. I won't do it in this PR as it will muddy the water